### PR TITLE
Add info on adding HA support for HPA w/disclaimer

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- This is requested by Solutions, customer referenced a [Blog](https://www.datadoghq.com/blog/autoscale-kubernetes-datadog/#enable-highly-available-support-for-hpa) post to set up High Availability for their HPA. But did not realize that they need to set up multiple orgs which doubles their bill.
- Request to add this enabling feature to the docs with a disclaimer that this impact billing
[DOCS-6575](https://datadoghq.atlassian.net/browse/DOCS-6575)

### Merge instructions

- [x] Ready to merge after review

[DOCS-6575]: https://datadoghq.atlassian.net/browse/DOCS-6575?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/agent/configuration/dual-shipping.md [hunk 1]
```diff
@@ -141,6 +141,20 @@ clusterAgent:
     - name: DD_ADDITIONAL_ENDPOINTS
       value: '{"https://app.datadoghq.com": ["apikey2"]}'
 ```
+### Cluster Agent metrics provider
+
+To ensure autoscaling is resilient to failure, configure the Cluster Agent to selectively fetch any metrics you use for HPA (both Kubernetes and standard/custom application metrics) from multiple Datadog regions. Configure the Datadog Cluster Agent manifest with several endpoints:
```
Comment (buraizu) [inline content/en/agent/configuration/dual-shipping.md:146]:
```suggestion
To ensure autoscaling is resilient to failure, configure the Cluster Agent to selectively fetch any metrics you use for HPA (both Kubernetes and standard or custom application metrics) from multiple Datadog regions. Configure the Datadog Cluster Agent manifest with several endpoints:
```

Comment (buraizu) [reply content/en/agent/configuration/dual-shipping.md:146]: I'm not totally clear on what `standard` means in this case -- is that any default Datadog or integration metric, or is it a type of application metric?

File: content/en/containers/guide/cluster_agent_autoscaling_metrics.md [hunk 2]
```diff
@@ -163,6 +163,8 @@ Once enabled, the Cluster Agent is ready to fetch metrics for the HPA. There are
 
 Datadog recommends using the `DatadogMetric` option. While this does require an additional step of deploying the `DatadogMetric` CustomResourceDefinition (CRD), this provides a lot more flexibility in the queries performed. If you do not use `DatadogMetric` queries, your HPAs use the native Kubernetes external metrics format, which the Cluster Agent translates into a Datadog metric query.
 
+If you are dual-shipping your metrics to multiple Datadog organizations, you can configure the Cluster Agent to fetch from these multiple endpoints for high-availability. For more information, see the [Dual Shipping][5] documentation.
```
Comment (buraizu) [inline content/en/containers/guide/cluster_agent_autoscaling_metrics.md:166]:
```suggestion
If you are dual-shipping your metrics to multiple Datadog organizations, you can configure the Cluster Agent to fetch from these multiple endpoints for high availability. For more information, see the [Dual Shipping][5] documentation.
```

**Review by buraizu — APPROVED** [review]
Looks good overall, approving with some minor feedback

File: content/en/agent/configuration/dual-shipping.md (hunk 1 above)
Comment (estherk15) [reply content/en/agent/configuration/dual-shipping.md:146]:
```suggestion
To ensure autoscaling is resilient to failure, configure the Cluster Agent to run your metric queries for the HPA against your multiple Datadog regions with dual-shipped data. Configure the Datadog Cluster Agent manifest with several endpoints:
```

**Pull request merged**